### ISSUE TITLE
Results for HPO examples

### ISSUE THREAD

**Issue by shikhardev** [wandb/examples#11]
Is it possible to post the results of performance of the different hyperparameter optimization experiments? Specifically, I am interested in pytorch cifar10, fashion and mnist experiments. The results could be the loss plots over time, the final accuracy received after specified budget, etc.

**Comment by vanpelt** [comment]
Sure, if you use the examples in this rep, you can create a [sweep](https://docs.wandb.com/sweeps/quickstart) to see the results!

**Comment by vanpelt** [comment]
Sorry, reading this again I believe you're asking to see existing visualizations for sweeps that have been run.  Here's one with fashion mnist: https://app.wandb.ai/shawn/fasion-sweep
